test/python/test_ibmqjob.py

```diff
@@ -57,11 +53,11 @@ def setUp(self, qe_token, qe_url):
         qc.cx(qr[0], qr[1])
         qc.measure(qr, cr)
         self._qc = qc
-        self._provider = IBMQProvider(qe_token, qe_url)
-        self._local_provider = LocalProvider()
```

Comment:
This line was inserted here: https://github.com/Qiskit/qiskit-terra/pull/686/files#r210262725 and it can be removed.